proposals/0004-hyperdb.md

```diff
@@ -474,16 +491,18 @@ The overall bytestring would be:
 ## Simple Put and Get
 
 Starting with an empty hyperdb `db`, if we `db.put('/a/b', '24')` we expect to
-see a single `Entry` and index 0:
+see a single `Entry` and index 1 (following the DEP-0007 hypercode header at
+index 0):
```

Comment:
This sentence is confusing to me. What are we supposed to see at index 1. What at index 0? I know because I read the Hypercore DEP, but it's not immediately clear from the text.

Reply:
Also, what's a `hypercode`? :smiley_cat: